test/Generator/fixtures/expected/KeyRegistry.php

```diff
@@ -0,0 +1,41 @@
+<?php
+// HEADER
+
+namespace Hostnet\Component\AccessorGenerator\Generator\fixtures\Generated;
+
+/**
+ * This class holds all the keys for the Traits in the same directory. It also
+ * provides methods to statically add keys (useful when testing).
+ */
+class KeyRegistry
+{
+    private static $public_key_paths = [
+       'database.table.column' => '',
+       'database.table.column_again' => '',
+    ];
+
+    private static $private_key_paths = [
+       'database.table.column' => '',
+       'database.table.column_again' => '',
+    ];
+
+    public static function getPublicKeyPath($alias)
```

Comment:
maybe also generate some docs? Seems rather usefull when you are going to use this

Other than that: LGTM :+1: